ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyOptimizer.java

```diff
@@ -342,8 +342,8 @@ private void applyLayoutBase(ReactShadowNode node) {
     // We use screenX/screenY (which round to integer pixels) at each node in the hierarchy to
     // emulate what the layout would look like if it were actually built with native views which
     // have to have integral top/left/bottom/right values
-    int x = node.getScreenX();
-    int y = node.getScreenY();
+    float x = node.getLayoutX();
+    float y = node.getLayoutY();
```

Comment:
You should update the comment as well

Reply:
I have update the comment, thank you

Reply:
@ecreeth I have update the comment and I submitted in the original branch. Do I need to launch a new PR?

Reply:
@ecreeth Can you review my commit again?